### DISCUSSION TITLE
Data Not Displayed When Clicking on Bootstrap Examples Components on Bootstrap's Website

### DISCUSSION THREAD

**Discussion by ot-net** [twbs/bootstrap#39473] [Q&A] [answered]
When I click on any component on the Bootstrap examples page (https://getbootstrap.com/docs/5.3/examples/), no data is displayed. I have tested this issue on Opera, Chrome, and Firefox browsers, and the problem persists.

![Screenshot 2023-12-06 at 11 09 07 AM](https://github.com/twbs/examples/assets/92269361/ee0016a8-e96e-4f3c-a739-c2a6c00434b3)

I would appreciate any assistance in resolving this issue. Thank you!

**Comment by julien-deramond** [comment]
Hello @goktugerol-dev 
I'm not sure to get what the issue is here. Apparently, you're in https://getbootstrap.com/docs/5.3/examples/footers. This page only shows the rendering of some footers that can be created using Bootstrap, but clicking on links is not supposed to do anything.

**Reply by ot-net** [reply, marked as answer]
I was expecting to see a variety of footer samples to select from. Initially, I thought they were not loading properly, but upon closer inspection, I realized that there aren't many samples available. The ones that are present seem to blend in with the webpage and made me think as a part of the webpage, rather than standing out as distinct sample designs. (I'm using dark mode on a Mac with the Opera GX browser.) So, that prompted me to open this issue. Closing the issue now.
